server/pom.xml

```diff
@@ -349,7 +349,7 @@
                                 <argument>-package</argument>
                                 <argument>com.broadcom.usecase</argument>
                                 <argument>-o</argument>
-                                <argument>target/generated-sources/antlr4/com/broadcom/usecase</argument>
+                                <argument>${basedir}/target/generated-sources/antlr4/com/broadcom/usecase</argument>
```

Comment:
Yes, or even:
```
${project.build.directory}/generated-sources/antlr4/com/broadcom/usecase
```

Reply:
${basedir} was there before, I just didn't know why it should be there. I thought it is some obsolete value

Reply:
it has to be there for build